obelix/idefix/services/scaleway/client.go

```diff
@@ -0,0 +1,110 @@
+package scaleway
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/Cleopha/ifttt-like-common/credentials"
+	"github.com/Cleopha/ifttt-like-common/protos"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/structpb"
+	"idefix/operator"
+	"strings"
+)
+
+var (
+	ErrInvalidScalewayCredentialsFormat = errors.New("invalid scaleway credentials format")
+)
+
+type Client struct {
+	ctx      context.Context
+	clt      *scw.Client
+	Operator *operator.IdefixOperator
+}
+
+func NewClient(ctx context.Context, op *operator.IdefixOperator) *Client {
+	return &Client{
+		ctx:      ctx,
+		clt:      nil,
+		Operator: op,
+	}
+}
+
+func (c *Client) configure(owner string) error {
+	credentialClient, err := credentials.NewClient("9001")
```

Comment:
think to get in env

Reply:
Fix in https://github.com/Cleopha/ifttt-like/pull/74/commits/ab007719ebd4e8eb7ced0110127319a56eef03cd